src/layers/core/hexagon-layer/hexagon-layer.js

```diff
@@ -77,9 +103,16 @@ export default class HexagonLayer extends Layer {
       const {viewport} = this.context;
 
       const hexagons = hexagonAggregator(this.props, viewport);
-      const countRange = _getCountRange(hexagons);
+      const sortedCounts = new SortedBins(hexagons);
```

Comment:
The name of this class doesn't feel quite right, but not sure I can't come up with anything better. `BinSorter`?